commands/intensify.js

```diff
@@ -0,0 +1,74 @@
+const Command = require('./command.js')
+const Image = require('../models/image.js');
+const ImageLoader = require('../util/image-loader.js')
+const ImageUploader = require('../util/image-uploader.js')
+const UtilArray = require('../util/array.js')
+const {TRANSPARENT_BLACK} = require('../constants.js')
+
+class Intensify extends Command {
+    static help() {
+        return `*Usage:* \`emojitools intensify <url>\``
+    }
+
+    async render() {
+        const image = await ImageLoader.fromUrl(this.getUrl());
+        const frame_delay = 5;
+
+        const frames = image['_frames'];
+
+        let expanded_and_split_frames = UtilArray.flatmap(frames.map((frame, idx) => {
+            const expandedFrame = this.expandFrame(frame);
+            return this.splitFrame(expandedFrame, frame_delay);
+        }));
+
+        let last_offset;
+        const intensified_frames = expanded_and_split_frames.map((frame, index) => {
+            last_offset = this.getOffset(last_offset);
+            const [offset_x, offset_y] = last_offset;
+            return this.intensifyFrame(frame, offset_x, offset_y);
+        });
+
+        return ImageUploader.upload(new Image(intensified_frames));
+    }
+
+    // Temporally split into subframes based on existing delay + new delay. For
+    // now we're going to ignore remainders.  TODO: more math?
+    splitFrame(frame, frame_delay) {
```

Comment:
it might make sense to put this on the `Frame` base class as something like `splitWithDelay`